pkgs/tools/package-management/nix/default.nix

```diff
@@ -181,12 +181,12 @@ in rec {
 
   nixUnstable = lib.lowPrio (callPackage common rec {
     name = "nix-2.3${suffix}";
-    suffix = "pre6779_324a5dc9";
+    suffix = "pre6779_84de821";
```

Comment:
Shouldn't 6779 be changed?

Reply:
Where does that part of the suffix come from, looking at the source there doesn't seem to be a reference to `6779` anywhere in either commit?

Reply:
`git rev-list $revision | wc -l`

Reply:
Done, thanks for help.